nengo_gui/components/ace_editor.py

```diff
@@ -6,18 +6,18 @@
 import nengo_gui.monkey
 
 class AceEditor(Component):
-    def __init__(self, viz, config, uid):
-        super(AceEditor, self).__init__(viz, config, uid)
-        self.viz = viz
+    def __init__(self, sim, config, uid):
+        super(AceEditor, self).__init__(sim, config, uid, z_order=-8)
```

Comment:
Any reason in particular for `z_order=-8`? Was there a problem with it being placed on top of stuff?